### PR TITLE
DOC: add list of side-packages

### PR DESCRIPTION
**Pull request by drammock**
per discussion on MNE_Analysis email list, this adds a list of "side-packages" to the installation docs.

### CODE REVIEW

**Comment by cbrnr** [comment]
Can we add MNELAB in this list?

File: doc/install/pre_install.rst [hunk 1]
```diff
@@ -28,10 +28,23 @@ Overview of the MNE tools suite
...
-`Brain Imaging Data Structure`_ specification (`MNE-BIDS`_).
+There is also a growing ecosystem of other Python packages that work alongside
+MNE-Python, including packages for:
+
+- easily importing MEG data from the Human Connectome Project for
+  use with MNE-Python (`MNE-HCP`_)
+- managing MNE projects so that they comply with the `Brain
+  Imaging Data Structure`_ specification (`MNE-BIDS`_)
+- automatic bad channel detection and interpolation (`autoreject`_)
+- sparse dictionary learning (`alphaCSC`_)
```
Comment (agramfort) [inline doc/install/pre_install.rst]:
```suggestion
- convolutional sparse dictionary learning and waveform shape estimation (`alphaCSC`_)
```

File: doc/install/pre_install.rst [hunk 2]
```diff
@@ -28,10 +28,23 @@ Overview of the MNE tools suite
...
+- preconditioned ICA for real data (`PICARD`_)
```
Comment (agramfort) [inline doc/install/pre_install.rst]:
```suggestion
- independent component analysis (ICA) with good performance on real data (`PICARD`_)
```

**Comment by drammock** [comment]
Oops! Yes of course, feel free to push that addition, or I can do it tomorrow.

File: doc/install/pre_install.rst [hunk 3]
```diff
@@ -28,10 +28,23 @@ Overview of the MNE tools suite
...
+- phase-amplitude coupling (`pactools`_)
+- representational similarity analysis (`rsa`_)
+- microstate analysis (`microstate`_)
+- connectivity analysis using dynamic imaging of coherent sources (`conpy`_)
```
Comment (agramfort) [inline doc/install/pre_install.rst]:
```suggestion
- connectivity analysis using dynamic imaging of coherent sources (DICS) (`conpy`_)
```

**Pull request merged**